### PR TITLE
V2.4-Trident-Spool-Holders

### PR DESCRIPTION
**Pull request by elcrni**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?

2 version of the V2.4-Trident-Spool-Holders, for printers that are backed closed to the wall.

#### How was it tested? 
Tested with 3 different printers and one Voron 2.4, works as expected.

#### Any background context you want to provide?
The need for this mod came from the fact that my 2.4 350mm2 was large and i had to back it up against the wall, there was no more room to mount the spool as in original design.

### CODE REVIEW

**Comment by cbc02009** [comment]
Is this compatible with v2.4r2?

**Comment by elcrni** [comment]
> 

Yes, it is.

**Review by cbc02009 — CHANGES_REQUESTED** [review]
please remove the 3mf files. they are not allowed in usermods.

**Comment by elcrni** [comment]
> please remove the 3mf files. they are not allowed in usermods.

.3mf files replaced with STLs. thanks

**Comment by cbc02009** [comment]
Maybe you pushed to the wrong branch? I'm still seeing them.

**Comment by elcrni** [comment]
> Maybe you pushed to the wrong branch? I'm still seeing them.

Wrong branch, sorry, should be ok now. thanks

**Review by cbc02009 — CHANGES_REQUESTED** [review]
Please fix the orientation of the two ball bearing arms. they're not flat on the bed.

**Comment by elcrni** [comment]
> Please fix the orientation of the two ball bearing arms. they're not flat on the bed.

fixed. thanks

**Review by cbc02009 — APPROVED** [review]

**Review by cbc02009 — CHANGES_REQUESTED** [review]
sorry, I just realized you forgot to add the mod to the readme.md in the base printer_mods folder.

**Comment by elcrni** [comment]
> sorry, I just realized you forgot to add the mod to the readme.md in the base printer_mods folder.

sorry, done now. thanks

File: printer_mods/README.md [hunk 1]
```diff
@@ -349,3 +349,4 @@ like so:
 | | [V0 Skirt set](./Zen3D/V0_Skirt)| SSR, PowerSwitch, Adxl, Nameplate and color change printing  | :heavy_check_mark: |:x: |:x: |:x:|:x: |
 | | [V0 1515 T-nut](./Zen3D/V0_1515_T-nut)| M3 square and hex nuts for Voron0 1515 frame | :heavy_check_mark: |:x: |:x: |:x:|
 | zruncho | [V0 No-Drop Nuts](./zruncho/V0_No_Drop_Nuts) | Voron0 nut inserts which auto-align (LDO frames only) | :heavy_check_mark: | :x: | :x: | :x: |
+| elcrni | [V.24/Trident Spool Holder](./elcrni/V2.4-Trident-Spool-Holders) | Spool holder that mounts on top for "too close to the wall" printers | :x: | :heavy_check_mark: | :heavy_check_mark: | :x: | :heavy_check_mark:|
```
Comment (cbc02009) [inline printer_mods/README.md:352]: Please add your entry in alphabetical order.

**Review by cbc02009 — CHANGES_REQUESTED** [review]

File: printer_mods/README.md (hunk 1 above)
Comment (elcrni) [reply printer_mods/README.md]: Done!

**Review by cbc02009 — APPROVED** [review]

**Pull request merged**